code-challenges.js

```diff
@@ -14,36 +29,139 @@ const fruit2 = "banana"
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
+const longerString = (stringOne, stringTwo) => {
+  if (stringOne.length > stringTwo.length) {
+    return stringOne
+  } else {
+    return stringTwo
+  }
+}
+console.log(longerString(fruit1, fruit2));
+console.log(longerString(fruit3, fruit4))
+
+
+// 1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-// --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+//------------------ PSEUDO CODE PORTION -----------------//
+//create a function
+//must take a number as a parameter
+// determine 3 different conditions by using if statement
+// if 42 is inputted, return should be set it to below boiling point
+// if 350 is inputted, return should be set it to above boiling point
+// if 212 is inputted, return should be set it to at boiling point
+//console log by referring to the function name and pass it a number as argument, in this case, i utilized the 3 variables provided.
+// -------------------------------------------------------//
+
+
+// ******************* CODE BELOW ******************* //
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const boilingPoint = (num) => {
+  if (num === 212) {
+    return `${num} is at boiling point.`
+  } else if (num >= 350) {
```

Comment:
Great job on getting a working function. Since the boiling point is 212, then the next appropriate conditionals would be (num > 212) for above boiling point and (num < 212) for below boiling point. The variables are testing all the conditionals. Looking over the question...it specifies that parameter. `Boiling point is 212 degrees Fahrenheit.` Other than that, quality work!